SPECS/kernel-hci/kernel-hci.spec

```diff
@@ -431,6 +433,9 @@ ln -sf linux-%{uname_r}.cfg /boot/mariner.cfg
 %{_sysconfdir}/bash_completion.d/bpftool
 
 %changelog
+* Mon Aug 07 2023 Lanze Liu <lanzeliu@microsoft.com> - 5.15.123.1-2
+- Add patch (0028) to enable DM multipath Kernel configurations
```

Comment:
Please specify the specific CONFIG options are are enabling/changing/disabling here.